src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@3.2.0/fonts/remixicon.css" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="header">
+            <h3 class="header_title hello">Today's TO Do </h3>
+            <button><i class="ri-loop-right-line"></i></button>
+        </div>
+        <div class="todo_main">
+            <form id="js-todo-form">
+                <input type="text" id="input" placeholder="Create a new todo .....">
+            </form>
```

Comment:
Template
![image](https://user-images.githubusercontent.com/78251927/235942419-69967bcb-a908-4f9d-a095-4bb4f8a3b837.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/235943349-635605ef-89b2-4b75-9c76-cd70e7d67340.png)

According to the project requirement, it is important that our design is similar to the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). 

![image](https://user-images.githubusercontent.com/78251927/235740491-7fe3ce92-bb04-4085-8055-a10bb13c9e1a.png)

Please kindly add the missing icon in your design 💪